text-to-speech/versta/export/__main__.py

```diff
@@ -97,18 +126,23 @@ def main(
     # Step 4: Save the tokenizer files
     tokenizer_files = save_tokenizer(converted_dir, output_dir)
 
-    # Step 5: Get all voices from voices directory
-    voices = get_voices(output_dir / "voices")
+    # Step 5: Get all voices from voices directory (for Kokoro) or handle Piper voices
+    if model_format == "kokoro":
+        voices = get_voices(output_dir / "voices")
+    else:
+        # Piper models don't use separate voice files like Kokoro
+        voices = []
 
     # Step 6: Create metadata file for the model
-    generate_metadata(version, output_dir, model, ["Kokoro"], ort_files, tokenizer_files, voices)
+    model_architectures = ["Piper"] if model_format == "piper" else ["Kokoro"]
```

Comment:
Move this inside a private helper function inside the `metadata.py` file instead. Keep the `__main__.py` file clean.